book/src/bridge/extern_rustqt.md

```diff
@@ -131,24 +132,32 @@ Where `<Property>` is the name of the property.
 
 These setters and getters assure that the changed signal is emitted every time the property is edited.
 
-It is also possible to specify custom getters, setters and notify signals, using flags passed like so:
+### Custom Properties
+
+In case the automatically generated functions do not work for your use-case, you can disable CXX-Qts auto-generation and write a totally custom property.
+For example, this could be the case if your property doesn't correspond to any single field in the inner Rust struct.
+
+You can specify custom getters, setters and notify signals, using flags passed like so:
 `#[qproperty(TYPE, NAME, READ = myGetter, WRITE = mySetter, NOTIFY = myOnChanged)]`
-> Note: the key for the flags use all capitals like in the Qt version of qproperty
+> **📝 Note**: the key for the flags use all capitals like in the Qt version of qproperty
 
-It is also possible to use any combination of custom functions or omit them entirely, but if flags are specified, read must be included as all properties need to be able to be read.
+It is possible to use any combination of flags or omit some of them entirely, but if any flags are specified, the `READ` flag must be included.
 
-Using the read flag will cause CXX-Qt to generate a getter function with an automatic name based off the property. e.g. `#[qproperty(i32, num, READ)]` will have a getter function generated called get_num in Rust, and getNum in C++.
+If a custom function is specified for a flag, the function must be declared in the bridge and a corresponding implementation must exist.
 
-If a custom function is specified, an implementation both in qobject::MyObject and and export in the bridge is expected.
+Any flag may be passed without specifying a function (e.g. `READ` instead of `READ=...`).
```

Comment:
Any is not correct? Eg `RESET` must have a function specified we have no auto generation option?

And note others like CONSTANT/REQUIRED/FINAL do not have a function